test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,82 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+import java.util.Collection;
+import java.util.ArrayList;
+import java.lang.Integer;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void isAnyNull_multipleArgumentsNull_trueReturned() {
+        assertTrue(Utils.isAnyNull(null, null, null, null));
+    }
+    
+    @Test
+    public void isAnyNull_multipleArgumentsObjects_falseReturned() {
+        assertFalse(Utils.isAnyNull(new Object(), new String(), new Object()));
+    }
+    
+    @Test
+    public void isAnyNull_multipleArgumentsNullAndObjects_trueReturned() {
+        assertTrue(Utils.isAnyNull(new Object(), new String(), null, null));
+    }
+    
+    @Test
+    public void isAnyNull_emptyArgument_falseReturned() {
+        assertFalse(Utils.isAnyNull());
+    }
+    
+    @Test (expected = NullPointerException.class)
+    public void isAnyNull_singleArgumentNull_nullPointerExceptionThrown() {
+        Utils.isAnyNull(null);
+    }
+
+    @Test
+    public void elementsAreUnique_differentClassesElements_trueReturned() {
+        Collection<Object> items = new ArrayList<Object>();
+        items.add(new Object());
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.